include/afl-fuzz.h

```diff
@@ -260,7 +260,7 @@ enum {
 
   /* 00 */ PY_FUNC_INIT,
   /* 01 */ PY_FUNC_FUZZ,
-  /* 02 */ PY_FUNC_post_process,
+  /* 02 */ PY_FUNC_POST_PROCESS,
```

Comment:
You still have to update all locations where you have used the lowcase letter version.
Always, always compile before committing